src/python/install.sh

```diff
@@ -454,6 +454,20 @@ if [[ "${INSTALL_PYTHON_TOOLS}" = "true" ]] && [[ $(python --version) != "" ]];
             echo "${util} already installed. Skipping."
         fi
     done
+    
+    # Temporary: Removes “setup tools” metadata directory due to https://github.com/advisories/GHSA-r9hx-vwmv-q579
+
+    VULNERABLE_VERSIONS=("3.10" "3.11")
+    RUN_TIME_PY_VER_DETECT=$(python --version 2>&1)
+    PY_MAJOR_MINOR_VER=${RUN_TIME_PY_VER_DETECT:7:4};
+
+    for vv in "${VULNERABLE_VERSIONS[@]}"; do
```

Comment:
Curious, why do we need a `for` loop? Instead, can we a conditional check and see if `PY_MAJOR_MINOR_VER` is within `VULNERABLE_VERSIONS`?

Reply:
changed as requested.